### DISCUSSION TITLE
Are these the expected output of airflow webserver -D and airflow scheduler -D?

### DISCUSSION THREAD

**Discussion by twointum** [apache/airflow#21533] [Q&A] [answered]
I am not seeing the webserver running anymore, I am wondering if there is any troubleshooting I can do with the information provided in the output after running these statements:

![image](https://user-images.githubusercontent.com/35749928/153683712-8564d009-bee7-43a1-b10d-04a3a77ba319.png)

I also don't see the ports running for Airflow:
![image](https://user-images.githubusercontent.com/35749928/153684063-b49e8b43-3189-40a0-905d-cfd78b1983fe.png)

This stuff used to run just fine but I did make some config changes (added a fernet key, changed time zone). I don't think it has anything to do with the issues now, but I included that info just in case. 

Thank you everyone!

**Comment by potiuk** [comment]
Try to run withough -D flag or look in the scheduler logs (you wil find them in ${AIRFLOW_HOME}/logs or ${HOME}/airflow/logs if you have no AIRFLOW_HOME defined. The -D flag is not supposed to print any information in the output because well ... the "-D" flag tells it to close all output and redirect everything to the log. This is what --deamonize flag is about.

You can find more info about the "usual deamonize" behaviour for example here https://stackoverflow.com/questions/3095566/linux-daemonize

**Comment by twointum** [comment]
Thank you that makes good sense. I don't see anything in the log file for the webserver however. Any ideas what file I'm supposed to open?

![image](https://user-images.githubusercontent.com/35749928/153689262-88773068-86d0-419f-b778-7d3b3d4eab1a.png)

**Reply by potiuk** [reply]
Just look in all logs. You should approach it all in the same way as you approach any running application of yours - Airflow is not different than any other app.

**Comment by twointum** [comment, marked as answer]
Thank you! The problem was:
lockfile.AlreadyLocked: /home/somakun/airflow/airflow-webserver-monitor.pid is already locked

And I needed to just delete both pid files in the directory. They were locking something. Thank you!
